content/en/docs/tutorials/verify.md

```diff
@@ -0,0 +1,184 @@
+
+```
+notation inspect $IMAGE
+```
+
+## Verifying a container image with Ratify 
+As a prerequisite, you’ll need:
+- Kubernetes v1.20 or higher (You can use [minikube](https://minikube.sigs.k8s.io/docs/start/) if you are new to Kubernetes)
+- Helm v3
+
+Using Ratify will ensure that only signed images are deployed to Kubernetes.
+Set up Gatekeeper with [external data.](https://open-policy-agent.github.io/gatekeeper/website/docs/externaldata)
+
+```console
+helm repo add gatekeeper https://open-policy-agent.github.io/gatekeeper/charts
+
+helm install gatekeeper/gatekeeper  \
+   --name-template=gatekeeper \
+   --namespace gatekeeper-system --create-namespace \
+   --set enableExternalData=true \
+   --set validatingWebhookTimeoutSeconds=5 \
+   --set mutatingWebhookTimeoutSeconds=2
+```
+Install the lastest version of Ratify. Specify the certificate generated by Notation for verification purposes.
+
+```console
+helm repo add ratify https://deislabs.github.io/ratify
+
+helm install ratify \
+   ratify/ratify --atomic \
+   --namespace gatekeeper-system \
+   --set-file notaryCert=ghcr-networks.io.crt
+```
+Apply the constrait to enforce Gatekeeper policy to allow only signed images can be deployed on Kubernetes:
+
+```console
+kubectl apply -f https://deislabs.github.io/ratify/library/default/template.yaml
+kubectl apply -f https://deislabs.github.io/ratify/library/default/samples/constraint.yaml
+```
+Deploy the sample image signed by Notation. Ratify will verify if this image has a valid signature.
+
+```console
+kubectl run ratify-signed --image=$IMAGE
+/net-monitor@sha256:27c0290c485140c3c998e92c6ef23fba2bd9f09c8a1c7adb24a1d2d274ce3e8e
+pod/demo created
+```
+Check the signature associated with the image.
+
+```
+notation ls $IMAGE
+```
+Deploy an unsigned image to Kubernetes cluster. The deployment has been denied since the image has not been signed and doesn't meet the deployment criteria.
+
+```console
+$ kubectl run demo --image=ghcr.io/feynmanzhou/notation/alpine@sha256:1304f174557314a7ed9eddb4eab12fed12cb0cd9809e4c28f29af86979a3c870
+```
+Output:
+```
+Error from server (Forbidden): admission webhook "validation.gatekeeper.sh" denied the request: [ratify-constraint] Subject failed verification: ghcr.io/feynmanzhou/notation/alpine@sha256:1304f174557314a7ed9eddb4eab12fed12cb0cd9809e4c28f29af86979a3c870
+```
+Inspect the logs to get the detailed error message from the Ratify Pod.
+
+```json 
+$ kubectl logs ratify-xxxx -n gatekeeper-system
+{
+     "subject": "ghcr.io/feynmanzhou/notation/alpine@sha256:1304f174557314a7ed9eddb4eab12fed12cb0cd9809e4c28f29af86979a3c870",
+     "isSuccess": false,
+     "message": "verification failed: no referrers found for this artifact"
+   }
+```
+Deploy a signed image with an expired certificate.
+
+```console
+kubectl run sample --image=ghcr.io/feynmanzhou/alpine:latest@sha256:1304f174557314a7ed9eddb4eab12fed12cb0cd9809e4c28f29af86979a3c870
```

Comment:
@FeynmanZhou and @Roseline-Bassey 

I understand the convenience of using a personal GHCR, but this really should be moved to some sort of official repo or the steps to recreate this artifact need to be listed. I don't think we should use personal repos in the docs.